microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -63,12 +63,12 @@ The following list provides update details:
 //[id="microshift-4-15-new-feat-based-on-{op-system-ostree}"]
 //==== Placeholder for new feat bases on RHEL Edge
 
-//[id="microshift-4-15-installation"]
-//=== Installation
+[id="microshift-4-15-installation"]
+=== Installation
 
-//[id="microshift-4-15-new-feature-for-use-at-installation"]
-//==== New feature for use during installation here
-//can include a note about a change in base RHEL image
+[id="microshift-4-15-blueprint-included"]
+==== Sample blueprints now included
+The `microshift-release-info` RPM now contains sample blueprints that can be used for image building. The blueprints include both {microshift-short} RPM packages and container image references.
```

Comment:
Do we need to show where the blueprint files are located, or is it up to the user to run the `rpm -ql <package>` to find them ?

Reply:
For a release note, we may want to skip the location to keep the text concise? 
In the docs, it should definitely be mentioned.

Reply:
@ggiguash That seems reasonable.

Reply:
No location in the release notes. We can xref back to the section in the install docs once we have that done. Remember that this part of the doc is headlines and highlights.